Controller/ControllerTrait.php

```diff
@@ -12,17 +12,23 @@
 namespace FOS\RestBundle\Controller;
 
 use FOS\RestBundle\View\View;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use FOS\RestBundle\View\ViewHandlerInterface;
 use Symfony\Component\HttpFoundation\Response;
 
 /**
- * Base Controller for Controllers using the View functionality of FOSRestBundle.
+ * Trait for Controllers using the View functionality of FOSRestBundle.
  *
  * @author Benjamin Eberlei <kontakt@beberlei.de>
+ * @author Lukas Kahwe Smith <smith@pooteeweet.org>
  */
-abstract class FOSRestController extends Controller
+trait ControllerTrait
```

Comment:
well the trait falls back to using the magic ``get()`` method .. but it can work without it .. 
I am not quite sure how to best approach this .. maybe provide two traits .. ?